dispatcher/conf/magento-proxy.conf

```diff
@@ -24,5 +24,3 @@ Define magento_host http://my-magento.cloud
 
 ProxyPass "/magento/graphql" "${magento_host}/graphql"
 ProxyPassReverse "/magento/graphql" "${magento_host}/graphql"
-ProxyPass "/magento/img" "${magento_host}/media/catalog/product"
```

Comment:
This two lines must not be changed !!! This is needed for images.

Reply:
Could you please enlighten me where /magento/img endpoint is needed for images? I haven't seen any use of it yet.